action.yml

```
@@ -11,18 +11,24 @@ inputs:
   token:
     description: Token with permissions to create and update pull requests
     required: true
-  synchronize-pull-request-template:
-    description: Whether to synchronize the pull-request template
-    default: true
-  synchronize-security-policy:
-    description: Whether to synchronize the security policy
-    default: true
   organization:
     description: Name of the repository's organization
     required: true
   repository:
     description: Name of the repository
     required: true
+  name:
+    description: Stylized name of the package
+    required: true
```

Does it make sense to add slightly more information in the description? For example 

Not a must have obviously.
```suggestion
  name:
    description: Stylized name of the package (MQT <...>)
    required: true
```

---

Yes, makes sense! I added examples now that should explain what is meant. 👍